### ISSUE TITLE
Simplify branching

### ISSUE THREAD

**Issue by soxofaan** [eu-cdse/documentation#629]
I made a couple of pull requests last week and was quite confused of the branching setup (e.g. I made my pull requests against `staging` branch instead of `publish` branch).

As far as I understand there are three permanent branches:
- `main`
- `publish`
- `staging`

Apparently everything from `publish` is automatically merged into `main`, but nothing else seems to happen on/with `main`, so I wonder what the point is of having both `main` and `publish`. The only actual difference between the two is that `main` is full of ugly merge commits.

It's also not clear what the purpose of `staging` branch is. The readme says
> Staging: Branch that contains the next release of the documentation portal
> ... other changes (than hotfixes) are made via a PR on the `staging` branch.

But how relevant is this still? I guess it's not, given that I made my PRs against the wrong branch (I used `staging` given the README, but it should have been `publish`)

In summary, I wonder why this setup is so complex. I think it can be simplified to standard workflow:
- `main` branch, which is used to publish from
- all the rest is done through feature branches and pull request, including "next release of doc portal"

**Comment by soxofaan** [comment]
bit more context about my confusion: https://github.com/eu-cdse/documentation/pull/613#issuecomment-2368086351

**Comment by soxofaan** [comment]
side-effect of the complexity: the "merge-publish-2-staging" job started failing : https://github.com/eu-cdse/documentation/actions/workflows/merge-publish-2-staging.yml
